### ISSUE TITLE
Add instructions on how to prune with GUI

### ISSUE THREAD

**Issue by erciccione** [monero-ecosystem/monero-GUI-guide#149]
https://monero.stackexchange.com/questions/11454/how-do-i-utilize-blockchain-pruning-in-the-gui-monero-wallet-gui

**Comment by ITwrx** [comment]
if it were me, i would probably just add: 

> when setting up your wallet, choose to "manually manage the daemon" (i can't remember exact terminology used in the setup wizard) and add "--prune-blockchain" to the daemon flags input, then start the daemon up. 

to the FAQ. I probably wouldn't bother with that whole stackexchange explanation for CLI stuff.

**Comment by erciccione** [comment]
Yeah i agree that's not necessary. A short line like the one you suggested would be enough.
